packages/admin/cms-admin/src/dam/DataGrid/FolderDataGrid.tsx

```diff
@@ -79,6 +79,55 @@ interface FolderDataGridProps extends DamConfig {
     selectionApi: ISelectionApi;
 }
 
+interface FolderDataGridToolbarProps {
+    id?: string;
+    breadcrumbs?: BreadcrumbItem[];
+    filterApi: IFilterApi<DamFilter>;
+    selectionApi: ISelectionApi;
+    hideArchiveFilter?: boolean;
+    additionalToolbarItems?: React.ReactNode;
+    uploadFilters: {
+        allowedMimetypes?: string[];
+    };
+}
+
+function FolderDataGridToolbar(props: FolderDataGridToolbarProps) {
```

Comment:
If you would destructure the props here, it would be easier to spot unused props 🤔